tests/phpunit/CRM/Mailing/TokensTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @group headless
+ */
+class CRM_Mailing_TokensTest extends \CiviUnitTestCase {
```

Comment:
yup I think you can just ignore this note ... because away from diversity thing you had talked about  ... it would time consuming to use PHPUnit_Framework_TestCase instead of CiviUnitTestCase for these tests .